.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,7 +6,7 @@ labels: 'enhancement'
 assignees: ''
 
 ---
-
+<!--  
```

Comment:
What are your thoughts on removing these?

Reply:
I would like to have the content as a comment and not as a template, so the user could uncomment only the part that they want. I'm missing some things?

Reply:
Thanks, makes sense! I thought it would just remove the message m, but a comment sounds good